### DISCUSSION TITLE
Use firebase backend

### DISCUSSION THREAD

**Discussion by albertocutone** [obytes/react-native-template-obytes#300] [General] [answered]
Hello,
I have a project which is using React Native, Expo and Firebase as backend. I came across the obytes template and I simply think is a great job. I would like to port my project and using your template, but I am worried about the firebase integration.
Do you think is feasible in an easy manner? What are the pros and cons?

**Comment by yjose** [comment, marked as answer]
The starter is simply a set of choices to begin a new Expo project and is primarily based on the Expo Dev Client. So, technically, there is no reason why you should have issues when trying to add Firebase. it's should be straight forward as easy as installing it on an expo project
